pastas/stressmodels.py

```diff
@@ -711,6 +711,10 @@ def __init__(self, stress, rfunc, name, distances, up=False, cutoff=0.999,
             raise NotImplementedError("WellModel only supports the rfunc "
                                       "HantushWellModel fow now!")
 
+        logger.warn("It is recommended to use LmFitSolve as the solver "
```

Comment:
Even when when changing the units of Q? If I divide by 1e6 there's often to problem.

Reply:
"to problem" -> no problem or a problem?

Reply:
The values of parameters A and b become even lower because the distance (`r`) is set to 1.

You can probably avoid this by changing units as you suggest, but then we'd need to recommend doing so somehow.

Reply:
I meant "no problem"